tests/e2e/app/authn-basic/index.html

```diff
@@ -8,6 +8,25 @@
   <meta name="referrer" content="unsafe-url">
 
   <link rel="shortcut icon" href="/fr-ico.png" type="image/png">
+
+  <style>
+    @media (prefers-color-scheme: dark) {
+      html {
+        background-color: black;
+        color: white;
+      }
+      a {
+        color: lightblue;
+      }
+      a:visited {
+        color: lavender;
+      }
+      a:hover {
+        color: lightskyblue;
+      }
+    }
+  </style>
+
```

Comment:
Just adding dark mode to the E2E tests for those that have dark mode set for the OS. The pure white page will not longer blind those that have dark mode enabled :)